openshift/aws-resource-exporter.yaml

```diff
@@ -52,6 +52,10 @@ objects:
             value: ${AWS_REGION}
           - name: AWS_RESOURCE_EXPORTER_CONFIG_FILE
             value: /etc/aws-resource-exporter/aws-resource-exporter-config.yaml
+          - name: ROLE_ARN
```

Comment:
ROLE_ARN should come from SECRET_NAME too. check https://console-openshift-console.apps.app-sre-stage-0.k3s7.p1.openshiftapps.com/k8s/ns/app-sre-observability-stage/secrets/aws-resource-exporter-cluster-scraper-app-sre. You may enable `optional: true` in case of it does not exist. https://stackoverflow.com/questions/48208705/how-to-mark-secret-as-optional-in-kubernetes